_staff_members/thomas.md

```diff
@@ -0,0 +1,7 @@
+---
+name:  Thomas Moreau
+position: Senior research scientist
+image_path: https://avatars.githubusercontent.com/u/3321081?v=4
+website: https://tommoral.github.io/
+blurb: I am a researcher in the parietal team @ INRIA Saclay, since autumn 2019, working on unsupervised learning for time series and on deep learning methods applied to solving inverse problems. 
```

Comment:
```suggestion
blurb: I am a researcher in the MIND team @ INRIA Saclay, since autumn 2019, working on unsupervised learning for time series, bilevel optimization and on deep learning methods applied to solving inverse problems. 
```